GETTING-STARTED.md

```diff
@@ -1,166 +1,307 @@
 Getting Started
 ===============
+If you're having problems after the initial setup. Read the [reminders](#10-reminders-before-you-get-started)
+
 
 ## 1. vJoy
 
 Before running the Elite VR Cockpit you will need to install and configure [vJoy](https://sourceforge.net/projects/vjoystick/files/latest/download).
 
-Open up `vJoy Configure` and make sure you have the following configured:
+Open up `vJoy Configure` and make sure you have 2 vJoy Devices Configured:
+
+#### vJoy device 1 (default enabled) ####
+* Axis: make sure all axis are enabled
+* Buttons: Set to 32. Technically, you only need 8, but setting this higher will make it future-proof
+* POV/HAT switches: set to "4 Directions" and make sure there are 4 POVs configured
+* Enable all Force Feedback effects
+
+| vJoy Device 1                             |
+| :---------------------------------------- |
+| ![vJoy Device 1](Images/vJoyDevice1.png)  |
 
-* vJoy Devices: In addition to the default vJoy Device 1 you must also enable vJoy Device 2
-* Axis: To simplify things make sure all axis are enabled
-* Buttons (device 1): For now 8 buttons are enough, but you can future proof your setup by setting it to 32
-* POV/HAT switches (device 1): Configure POVs to "4 Directions" and make sure there are 4 POVs configured
+#### vJoy device 2 ####
+enable this by clicking the "2" tab at the top and then clicking the "Add Device" button near the bottom of the window
+* Axis: make sure all axis are enabled (same as device 1)
+* Buttons: Set to 32 (same as device 1)
+* POV/HAT switch: set to continuous and 0 POVs
```

Comment:
That's a fair point! I usually like to leave explicit instructions, in case defaults ever change. It doesn't seem likely in this case, but would you be ok if I just appended a (default) tag to the end of that line?

Reply:
I guess adding a (default) tag would be fine. But I'm saying the overlay doesn't use device 2's POVs at all. If the user changed the device 2 POVs to the non-default 4-way POVs, it wouldn't matter to the overlay. A "Leave unchanged" instead of a explicit instruction would be equally useful.

Reply:
Ah, that makes more sense. I've added a note to that effect

Reply:
I don't see any note on this in the current version.